eng/pipelines/backup_build_artifacts.yml

```diff
@@ -0,0 +1,102 @@
+# No CI trigger rely on Pipeline trigger only
+trigger: none
+
+resources:
+  pipelines:
+    - pipeline: officialbuild
+      source: NuGet.Client-Official
+      trigger: true
+
+jobs:
+  - job: Backup_Build_Artifacts
+    displayName: "Backup Build Artifacts"
+    timeoutInMinutes: 10
+    pool:
+      name: VSEngSS-MicroBuild2022-1ES
+
+    steps:
+      - checkout: none
+
+      - pwsh: |
+          $buildNumber = "$(resources.pipeline.officialbuild.runName)"
+          Write-Host "Setting build number to $buildNumber"
+          Write-Host "##vso[build.updatebuildnumber]$buildNumber"
+        displayName: "Set build number"
+
+      - pwsh: "Get-ChildItem Env: | Sort-Object Name | Format-Table -Wrap -AutoSize"
+        displayName: 'Display environment variables'
+
+      - task: DownloadPipelineArtifact@2
+        displayName: 'Download product NonRTM'
+        continueOnError: true # in case build didn't publish this artifact
+        inputs:
+          buildType: specific
+          project: $(resources.pipeline.officialbuild.projectID)
+          definition: $(resources.pipeline.officialbuild.pipelineID)
+          artifactName: 'VS15'
+          targetPath: $(Build.ArtifactStagingDirectory)/backup/VS15
+
+      - task: DownloadPipelineArtifact@2
+        displayName: 'Download product RTM'
+        continueOnError: true # in case build didn't publish this artifact
+        inputs:
+          buildType: specific
+          project: $(resources.pipeline.officialbuild.projectID)
+          definition: $(resources.pipeline.officialbuild.pipelineID)
+          artifactName: 'VS15-RTM'
+          targetPath: $(Build.ArtifactStagingDirectory)/backup/VS15-RTM
+
+      - task: DownloadPipelineArtifact@2
+        displayName: 'Download nupkgs'
```

Comment:
The rest of the steps have a nonRTM and RTM. 
To be consistent you can rename this to nonRTM

Reply:
good catch!